src/components/hand-controls.js

```diff
@@ -74,30 +117,75 @@ module.exports.Component = registerComponent('hand-controls', {
    * @param {string} evt the event associated to the button
    */
   handleButton: function (button, evt) {
-    var el = this.el;
+    // var el = this.el;
     var isPressed = evt === 'down';
+    var isTouched = evt === 'touchstart';
     switch (button) {
       case 'trackpad':
         if (isPressed === this.trackpadPressed) { return; }
         this.trackpadPressed = isPressed;
-        this.playAnimation('thumb', !isPressed);
-        evt = isPressed ? 'thumbup' : 'thumbdown';
-        el.emit(evt);
+        this.processAnimation();
```

Comment:
You can just call `this.animate()` at the end of the function instead of having the call on each case branch